packages/kit/src/runtime/server/page/load_node.js

```diff
@@ -101,6 +101,12 @@ export async function load_node({
 					};
 				}
 
+				//merge  headers  from  request
```

Comment:
we usually put a space after `//`
```suggestion
				// merge  headers  from  request
```

Reply:
There are two spaces between words too, we should have one space only.